tests/post/features/volume.feature

```diff
@@ -11,15 +11,6 @@ Feature: Volume management
 
     Scenario: Test volume creation (sparseLoopDevice)
         Given the Kubernetes API is available
-        When I create the following Volume:
-            apiVersion: storage.metalk8s.scality.com/v1alpha1
-            kind: Volume
-            metadata:
-              name: volume1
-            spec:
-              nodeName: bootstrap
-              storageClassName: metalk8s-prometheus
-              sparseLoopDevice:
-                size: 10Gi
+        When I create a Volume 'volume1'
```

Comment:
You should explain why we suddenly introduce an helper.

Something like:

```
test/volume: refactor volume creation test to use helpers 
In future test we will need to create Volume by giving only a name
(we will use a default template for the other fields) as well as
creating Volume from a YAML manifest (what we are already doing).

Since the creation code will be the same, let's factor in into an helper.
```